torch/_inductor/codegen/cuda/cutlass_epilogue_gen.py

```diff
@@ -0,0 +1,300 @@
+from typing import List
+from unittest.mock import patch
+
+import sympy
+
+import torch._inductor.virtualized as virtualized
+from torch._inductor.ir import ComputedBuffer, FlexibleLayout, IRNode, Pointwise
+from torch._inductor.utils import IndentedBuffer, sympy_str
+
+
+def _arg_str(a):
+    if isinstance(a, sympy.Expr):
+        return "sympy_expr('" + sympy_str(a) + "')"
```

Comment:
My guess is that in very special cases like these:

```
s = a.size(0)
c = torch.mm(a, b) * s
```

I think, this should work in PT Eager, but not sure if it will even reach the CUTLASS codegen in PT2 / Inductor.

Reply:
This is a remainder from KernelFormatterHandler code, I just overlooked that it's still in there. These would be mostly relevant if we were in a Triton Loop construct. I'm not sure if this might get relevant later when trying to fuse reductions or such, but at the moment ( for Pointwise ops ) this is not relevant. Will see if I can safely remove it.

Reply:
So, if any of these expressions turn up in the generated code, it indicates a problem. But it should not happen for Pointwise ops. Luckily I can just check pre-fusion and make sure it really never happens. Updating the code accordingly.